src/Data/AudienceFolder.elm

```diff
@@ -23,6 +30,44 @@ type alias AudienceFolder =
     }
 
 
+-- = BASIC FIELDS DECODERS =
+idField = D.field "id" D.int
+nameField = D.field "name" D.string
+parentField = D.field "parent" (D.nullable D.int)
+
+-- = AUDIENCE FOLDER DECODER --
+folderDecoder : Decoder AudienceFolder
+folderDecoder =
+  D.succeed AudienceFolder
+    |> D.andMap idField
+    |> D.andMap nameField
+    |> D.andMap parentField
```

Comment:
This eliminates my usual question of "what would you choose if you ran out of `map8`? :joy: Obviously given your background you know about applicatives. :+1: